modules/builds-image-source.adoc

```diff
@@ -60,12 +64,9 @@ This feature is not supported for builds using the custom strategy.
 ====
 endif::[]
 
-/////
-[role="_additional-resources"]
-.Additional resources
+.Images on mirrored registries that require pull secrets
 
-* Custom Strategy
-ifndef::openshift-online[]
-* ImageStreamTags
-endif::[]
-/////
+When using an input image from a mirrored registry, if you get a `build error: failed to pull image` message, you can resolve the error by using either of the following methods:
```

Comment:
```suggestion
When using an input image from a mirrored registry, if you get a `build error: failed to pull image` message, resolve the error by using either of the following methods:
```

Reply:
In this case, I prefer to reassert that the reader is the subject and that the action is optional. I'll be happy to explain why in person if you would like to discuss.